robottelo/constants.py

```diff
@@ -183,12 +184,33 @@
     'rhva6': (
         'Red Hat Enterprise Virtualization Agents for RHEL 6 Server (RPMs)'
     ),
+    'rheo7': 'Red Hat Enterprise Linux 7 Server - Optional (RPMs)',
+    'rhsc7': 'Red Hat Satellite Capsule 6.2 (for RHEL 7 Server) (RPMs)',
```

Comment:
Should we dynamically add the version number instead of hard code? Either that OR we will have to make sure that it gets updated once Satellite itself gets updated to 6.3. If this gets cherry-picked to 6.1.z, the same will have to be done.